[ADD] account_payment_pro_receiptbook: manual receiptbook numbering

jjscarafia opened the pull request.

jjscarafia committed:
[ADD] pro_receiptboo: manual receiptbook numbering

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/567).

jjscarafia committed:
[REM] _payment_pro: manual numbering formatting

Al final como en uruguay querian flexibilidad en la numeración y como era medio invasivo en versión estable crear nuevos tipos de documento y otros cambios.
Revertimos algunos cambios y los dejamos para más adelante.
De hecho, probablemente lo mejor sea dejar de usar tipos de documento ya que en realidad no los estamos usando para mucho (o dejarlos solo opcionales para formatear)
El prefijo de RE-X y OP-X debería de hecho ir en la misma secuencia o hardcodedao (similar a como hace odoo en los pagos 18+)

jjscarafia commented:
lo cerramos porque al final resuelven con numeración automática y secuencial el caso de uso
